src/core/utils/components/MiddleTextTruncation.tsx

```diff
@@ -47,6 +47,7 @@ export const MiddleTextTruncation = (props: MiddleTextTruncationProps) => {
       style={{
         display: 'flex',
         minWidth: 0,
+        flexGrow: 1,
```

Comment:
@veekeys Can you verify this is fine, since you implemented this component?

Reply:
Yeah, it is needed.. It is weird, why I dont have it, cause i was sure I added it. But it worked well without it in select examples somehow though... Might be the case only with the elements size change, though..